actors/miner/src/types.rs

```diff
@@ -145,6 +148,53 @@ pub struct ExpirationExtension {
     pub sectors: UnvalidatedBitField,
     pub new_expiration: ChainEpoch,
 }
+#[allow(clippy::too_many_arguments)] // validate mut prevents implementing From
+impl Into<ValidatedExpirationExtension> for ExpirationExtension2 {
+    fn into(mut self) -> ValidatedExpirationExtension {
+        let mut sectors = BitField::new();
+        for sc in self.sectors_with_claims {
+            sectors.set(sc.sector_number)
+        }
+        sectors |= self.sectors.validate_mut().unwrap();
+
+        ValidatedExpirationExtension {
+            deadline: self.deadline,
+            partition: self.partition,
+            sectors,
+            new_expiration: self.new_expiration,
+        }
+    }
+}
+
+#[derive(Serialize_tuple, Deserialize_tuple)]
+pub struct ExtendSectorExpiration2Params {
+    pub extensions: Vec<ExpirationExtension2>,
+}
+
+impl Cbor for ExtendSectorExpiration2Params {}
+
+#[derive(Serialize_tuple, Deserialize_tuple, Clone)]
+pub struct SectorClaim {
+    pub sector_number: SectorNumber,
+    pub maintain_claims: Vec<ClaimID>,
+}
+
+#[derive(Serialize_tuple, Deserialize_tuple)]
+pub struct ExpirationExtension2 {
+    pub deadline: u64,
+    pub partition: u64,
+    pub sectors: UnvalidatedBitField, // IDs of sectors without FIL+ claims
```

Comment:
These have all been changed to `Bitfield` in #617 (see #34), so do that here right away.

This might prompt merging master into our integration branch before landing this, since that change probably touches more of the code you've touched too.